sdk/communication/azure-communication-administration/tests/test_phone_number_administration_client_async.py

```diff
@@ -126,6 +128,20 @@ def setUp(self):
             self.capabilities_id = "capabilities_id"
             self.release_id = "release_id"
 
+    @AsyncPhoneNumberCommunicationTestCase.await_prepared_test
+    @pytest.mark.live_test_only
+    @pytest.mark.skipif(SKIP_PHONE_NUMBER_TESTS, reason=PHONE_NUMBER_TEST_SKIP_REASON)
+    async def test_list_all_phone_numbers_from_managed_identity(self):
+        endpoint, access_key = parse_connection_str(self.connection_str)
+        credential = create_token_credential()
+        phone_number_client = PhoneNumberAdministrationClient(endpoint, credential)
+        async with phone_number_client:
+            pages = phone_number_client.list_all_phone_numbers()
+            items = []
+            async for item in pages:
+                items.append(item)
+        assert len(items) > 0
```

Comment:
should we add assert to check for what we get in item? instead of just making sure the len(items) is not empty?

Reply:
Same as other comment.